### PR TITLE
Adding (working) SQL file and screenshot in readme

### PR DESCRIPTION
**Pull request by vistafan12**

### CODE REVIEW

**Comment by Raymonf** [comment]
Oh, there's even a PR! Can't accept this, though, since the SQL file is not complete. If you're still interested, feel free to fix that.

**Comment by vistafan12** [comment]
Hmm...

**Comment by Raymonf** [comment]
Hmm...

**Comment by vistafan12** [comment]
Hmm...!
http://updx.xyz/f/siv791.png

**Comment by vistafan12** [comment]
It's working!

File: osu.sql [hunk 1]
```diff
@@ -26,18 +28,30 @@ SET time_zone = "+00:00";
...
 --
 
-CREATE TABLE IF NOT EXISTS `scores` (
-  `maphash` varchar(32) NOT NULL,
-  `passed` tinyint(1) NOT NULL
+CREATE TABLE `scores` (
+  `maphash` varchar(900) NOT NULL,
+  `user` varchar(30) NOT NULL,
+  `submithash` varchar(900) NOT NULL,
+  `300s` longtext NOT NULL,
```
Comment (Raymonf) [inline osu.sql]: Could you maybe make the numerical columns integers?

**Pull request merged**